tests/sys/psa_crypto_hashes/main.c

```diff
@@ -41,6 +42,16 @@ int main(void)
         printf("Hash failed: %s\n", psa_status_to_humanly_readable(status));
     }
 
+
+#if IS_USED(MODULE_PSA_RIOT_HASHES_SHA_3)
```

Comment:
```suggestion
```

I would remove that so we could also test an hardware backend with the same code as soon as it is supported.